src/__tests__/App.test.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import App from '../index';
+import {Jsx} from "../index";
```

Comment:
Please explain the renaming...
Also single quotes not double

Reply:
I made the Jsx a component from the index.js file and was testing that it AppRouter was defined. Will push a PR with singe qoutes